components/controls/Tooltip.tsx

```diff
@@ -20,7 +20,8 @@
 import { throttle } from 'lodash';
 import * as React from 'react';
 import { createPortal, findDOMNode } from 'react-dom';
-import ScreePositionFixer from './ScreenPositionFixer';
+import ThemeContext from '../ThemeContext';
+import ScreenPositionFixer from './ScreenPositionFixer';
 import './Tooltip.css';
 
 export type Placement = 'bottom' | 'right' | 'left' | 'top';
```

Comment:
I know this is not part of the rework done here, but I'd rather see an enum here, so we avoid spreading string literals around the code. It is of course subject to discussion.

Reply:
Hmm, on second thought, although I'm in favor of this in principle, this component is used in so many places, we would need to refactor many, many calls, for very little gain. I'd leave it like this for now.